app/views/groups/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="group-nav d-flex flex-row justify-content-between">
+ <%= link_to  groups_path, class: "link-add-cat ps-2" do %>
+    <i class="fa-sharp fa-solid fa-arrow-left top-link"></i>
+  <% end %>
+  <h3 class="nav-add-tras">TRANSACTIONS</h3>
+  <%= link_to destroy_user_session_path, data: {turbo_method: :delete}, class: "link-add-cat pe-2 top-link me-3" do %>
+     <i class="fa-solid fa-arrow-right-from-bracket pe-2"></i>
+  <% end %>
+</div><br/>
+    <lu class="d-flex flex-column container-fluid">
+      <% @groups.each do |group| %>
+      <%= link_to group_path(group), class: "link-" do %>
+        <%= render "group", group: group%>
+        <% end %>
+      <% end %>
+    </lu>
+    <br/>
+<div class="container-fluid hom">
+  <div class="d-flex justify-content-center  btn-p">
+    <%= link_to "Add Item", new_group_path, class: "btn-prima" %>
+  </div>
+</div>
```

Comment:
- [OPTIONAL] Great work implementing all the changes suggested by the previous reviewers. It would be nice to make the _**Add Item**_ button occupy the full width of the page as it is positioned at the bottom of the page. This will improve the design of the button and make your design look very identical to that of the design template provided. 👍 😎 

### Your Design

![Screenshot1](https://user-images.githubusercontent.com/77106962/235185947-2f97d5c6-8c59-4ada-a9d3-0cd8b7ecf25d.png)

### Design Template

![image](https://user-images.githubusercontent.com/77106962/235186217-f64d8249-6e02-4123-bdb8-357d6f55a05f.png)

Reply:
@Gambit142  Thank You for your review, and suggestion